documentation/jetty-documentation/src/main/asciidoc/programming-guide/maven/jetty-maven-plugin.adoc

```diff
@@ -57,13 +57,15 @@ We recommend either the traditional link:{DISTGUIDE}[distribution] deployment ap
 [[get-up-and-running]]
 ==== Get Up and Running
 
-First, add `jetty-maven-plugin` to your `pom.xml` definition:
+Since Jetty 12, Jetty Maven plugin is repackaged for corresponding Jakarta EE version with an `eeX` classifier in the groupId and artifactId. The following examples in this document assume you are using Jakarta EE 10. 
+
+First, add `jetty-ee10-maven-plugin` to your `pom.xml` definition:
```

Comment:
same here we need something generic

Reply:
Yeah, so I add a small section for demonstrating Jakarta EE 10.

Reply:
use `ee{8,9,10}` instead